netlify.toml

```diff
@@ -0,0 +1,2 @@
+[build.environment]
+    NODE_VERSION = "16"
```

Comment:
This isn't the auto-generated file, right?
If yes, please add a new line to the end.

Reply:
No, I added it myself, I also added a line